integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -532,6 +599,12 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         Duration::seconds((7 * params.epoch_length) as i64),
     );
 
+    // At the end of the test we know for sure resharding has been completed.
+    // Verify that state is equal across tries and flat storage for all children shards.
+    let clients =
+        client_handles.iter().map(|handle| &test_loop.data.get(handle).client).collect_vec();
+    assert_state_sanity_for_children_shard(parent_shard_uid, &clients[0]);
+
```

Comment:
Would it make sense or be an overkill to to do at every height after the resharding block?